### ISSUE TITLE
Basic smilie support

### ISSUE THREAD

**Issue by mohlek** [KoKuToru/gTox#25]
How about a basic smilie support?

**Comment by Spagy** [comment]
All transforming of text sequences to emoji/unicode should happen in the sending client according to the [Tox STS on emoticons](https://github.com/Tox/Tox-STS/blob/master/STS.md#emoticons).

Custom-Emoji: "love" -> "❤️️" and ":)" -> "😊"

Simple example 1:
User types: "I love you :)"
gTox sends: "I ❤️️ you 😊"

gTox will render: "I ❤️️ you 😊"
Other STS Clients will also render the same

Simple example 2:
We recv: "I love you "

gTox will render: "I love you"
Other STS Clients will also render the same

Text sequence -> emoji/unicode should be optional though.

**Comment by KoKuToru** [comment]
Yeah "optional" is very important, it's stupid to just replace every ":)" with a "😊"

when I write ":)" i mean ":)"
when i write ":)" + something (key combo or whatever) I mean "😊" 

anyways I am going to use http://emojione.com/ for this..

**Comment by Spagy** [comment]
That makes sense but I think it would be good to have a setting to enable automatic emojifying.
